sdk/storagesync/Azure.ResourceManager.StorageSync/api/Azure.ResourceManager.StorageSync.netstandard2.0.cs

```diff
@@ -380,6 +382,12 @@ namespace Azure.ResourceManager.StorageSync.Models
         public static bool operator !=(Azure.ResourceManager.StorageSync.Models.ChangeDetectionMode left, Azure.ResourceManager.StorageSync.Models.ChangeDetectionMode right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class CloudEndpointAfsShareMetadataCertificatePublicKeys
```

Comment:
This name is weird, not sure if we can have a better name

Reply:
This name comes from another api (AfsShareMetadataCertificatePublicKeys API) and was mostly put in swagger for compliance. It won't be used by customers so it might not be worth it to change.